pages/quantum.js

```diff
@@ -53,139 +61,453 @@ import {
+            <div class="ComponentsContainer">
+              <Row className="Borders">
+                <Col className="Borders" xsmall={2} small={12}>
+                  12 Columns Width
+                </Col>
+              </Row>
+              <Row className="Borders">
+                <Col className="Borders" xsmall={2} small={6}>
+                  6 Columns Width
+                </Col>
+                <Col className="Borders" xsmall={2} small={6}>
+                  6 Columns Width
+                </Col>
+              </Row>
+              <Row className="Borders">
+                <Col className="Borders" xsmall={4} small={4}>
+                  4 Columns Width
+                </Col>
+                <Col className="Borders" xsmall={4} small={4}>
+                  4 Columns Width
+                </Col>
+                <Col className="Borders" xsmall={4} small={4}>
+                  4 Columns Width
+                </Col>
+              </Row>
+            </div>
+          </div>
+          <div class="RoundedEdges ComponentsContainer Width">
+            <div class=" WhiteBackground  RoundedEdges ComponentsContainer Width">
+              <h4 class="TextFormattingH4">Lists</h4>
+              <h6 class="TextFormattingH6">
+                List is a component with continuous, vertical indexes of texts
+                or icons.
+              </h6>
+              <List>
+                <List.Header>List Header</List.Header>
+                <br />
+                <List.SubHeader>List Subheaders</List.SubHeader>
+                <br />
+                <List.Content>
+                  <List.Item>List</List.Item>
+                  <br />
+                </List.Content>
+              </List>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="FlexContainer  ColoredBackground ComponentsContainer">
+        <div class="Width PaddingLeftRight">
+          <div class="PaddingTopBotttom  ComponentsContainer WhiteBackground RoundedEdges">
+            <h4 class="TextFormattingH4">Autocomplete</h4>
+            <h6 class="TextFormattingH6">
+              Styled input component with autocomplete suggestions
+            </h6>
+            <AutoComplete
+              suggestions={["AutoComplete suggestion"]}
+            ></AutoComplete>
+          </div>
+          <div class="PaddingTopBotttom ComponentsContainer WhiteBackground RoundedEdges">
+            <h4 class="TextFormattingH4">Dropdown</h4>
+            <h6 class="TextFormattingH6">
+              Styled dropwdown menu component following Catho's Style Guide.
+            </h6>
+            <Dropdown items={["ItemTest"]} />
+          </div>
+          <div class="PaddingTopBotttom ComponentsContainer WhiteBackground RoundedEdges">
+            <h4 class="TextFormattingH4">Input</h4>
+            <h6 class="TextFormattingH6">
+              Styled input component following Catho's Style Guide.
+            </h6>
+            <Input />
```

Comment:
how about of using all possible props of this components here?
With its labels and helper text?

Reply:
Removed since it appears again with Forms